run_email_parse.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+import re,sys
+import scrapy
+from scrapy.crawler import CrawlerProcess
+from scrapy.spiders import CrawlSpider, Rule
+from scrapy.linkextractors import LinkExtractor
+from scrapy import signals
+from scrapy.conf import settings
+from scrapy.xlib.pydispatch import dispatcher
+from selenium import webdriver
+
+settings.set('LOG_LEVEL', 'INFO')
+
+class EmailSpider(CrawlSpider):
+    name = 'email'
+
+      
+    def __init__(self, **kwargs):
+	self.allowd_domains = [kwargs['domain']]
+	self.start_urls= ["http://www."+kwargs['domain']]
```

Comment:
Should probably also check the input, otherwise this will prepend things twice. Also the subdomain might not always be www